packages/Manager/src/test/kotlin/nz/govt/eop/geo/GeoJsonControllerIntegrationTest.kt

```diff
@@ -26,4 +31,37 @@ class GeoJsonControllerIntegrationTest(@Autowired val mvc: MockMvc) {
     mvc.perform(get("/layers/whaitua").contentType(MediaType.APPLICATION_JSON))
         .andExpect(status().isOk)
   }
+
+  @Test
+  fun `can load surface water management units`() {
+    mvc.perform(
+            get("/layers/surface-water-management-units").contentType(MediaType.APPLICATION_JSON))
+        .andExpect(status().isOk)
+  }
+
+  @Test
+  fun `can load surface water management sub units`() {
+    mvc.perform(
+            get("/layers/surface-water-management-sub-units")
+                .contentType(MediaType.APPLICATION_JSON))
+        .andExpect(status().isOk)
+  }
+
+  @Test
+  fun `can load flow management sites`() {
+    mvc.perform(get("/layers/flow-management-sites").contentType(MediaType.APPLICATION_JSON))
+        .andExpect(status().isOk)
+  }
+
+  @Test
+  fun `can load flow limits`() {
+    mvc.perform(get("/layers/flow-limits").contentType(MediaType.APPLICATION_JSON))
+        .andExpect(status().isOk)
+  }
+
+  @Test
+  fun `can load groundwater zones`() {
+    mvc.perform(get("/layers/groundwater-zones").contentType(MediaType.APPLICATION_JSON))
+        .andExpect(status().isOk)
+  }
```

Comment:
Good to see a smoke test for each of these 👍

Reply:
Co-Pilot did most of those ... amazing!